testsuite/features/secondary/min_salt_migration.feature

```diff
@@ -0,0 +1,173 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+# This feature depends on 
+# - minkvm_guests.feature which has already configured the KVM Minion to be a virtualization host.
+# - srv_sync_channels.feature which syncs SLES 15 SP4 and the client tools.
+# We also test 'Bootstrapping using the command line' in this feature
+
+@scope_salt
+@scope_salt_ssh
+@scope_virtualization
+@virthost_kvm
+Feature: Migrate Salt to bundled Salt on a nested Minion VM
+
+  Scenario: Log in as admin user in the Salt migration context
+    Given I am authorized for the "Admin" section
+
+  Scenario: Prepare a SLES KVM test VM and list it in the Salt migration context
+    Given I am on the Systems overview page of this "kvm_server"
+    When I delete default virtual network on "kvm_server"
+    And I create salt-sles virtual network on "kvm_server"
+    And I delete default virtual storage pool on "kvm_server"
+    And I create salt-sles virtual storage pool on "kvm_server"
+    And I create a sles virtual machine named "sles-salt-migration" on "kvm_server"
+    And I follow "Virtualization" in the content area
+    And I wait until I see "sles-salt-migration" text
```

Comment:
As this is part of the preparation for the test, I wonder if we can do it via API/Command-line to speed up.

Reply:
I can look into that.

Reply:
Ok, only the last 2 steps are using the webUI the rest is already using the command line.

Reply:
Using the API for that ist not working properly:
![image](https://user-images.githubusercontent.com/12104291/203965457-a25579d2-e61c-4815-ae6c-76a6fc0ecea1.png)

Reply:
I reverted this back to the old state where the system gets deleted via the webUI. This is more stable atm. We can improve that or move more things to the command line/API in the future.